packages/hash/frontend/src/pages/[account-slug]/types/entity-type/property-expected-values.tsx

```diff
@@ -1,16 +1,10 @@
 import { PropertyType } from "@blockprotocol/type-system-web";
 import { Chip } from "@hashintel/hash-design-system/chip";
+import { types } from "@hashintel/hash-shared/types";
 
-const dataTypeNames = {
-  "https://blockprotocol.org/@blockprotocol/types/data-type/text/v/1": "Text",
-  "https://blockprotocol.org/@blockprotocol/types/data-type/number/v/1":
-    "Number",
-  "https://blockprotocol.org/@blockprotocol/types/data-type/boolean/v/1":
-    "Boolean",
-  "https://blockprotocol.org/@blockprotocol/types/data-type/null/v/1": "Null",
-  "https://blockprotocol.org/@blockprotocol/types/data-type/object/v/1":
-    "JSON Object",
-} as Record<string, string>;
+const dataTypeIdToTitle = Object.values(types.dataType).reduce<
+  Record<string, string>
+>((prev, { dataTypeId, title }) => ({ ...prev, [dataTypeId]: title }), {});
```

Comment:
I think this inline destructuring is really quite confusing. This feels a bit too much like clever code.

Reply:
That's fair, have made the destructuring more readable in https://github.com/hashintel/hash/pull/1193/commits/df9cf0301af0a3aabebba384d34c90c39cf7c73e

Reply:
Thanks :)